go/consensus/cometbft/apps/keymanager/churp/txs.go

```diff
@@ -545,3 +556,18 @@ func resetHandoff(status *churp.Status, nextHandoff beacon.EpochTime) {
 	status.NextChecksum = nil
 	status.Applications = nil
 }
+
+func verifyPolicy(ctx *tmapi.Context, policy *churp.SignedPolicySGX) error {
+	// Allow non-empty `MayQuery` field with the 24.2 release.
+	enabled, err := features.IsFeatureVersion(ctx, "24.2")
```

Comment:
We should also prepare an upgrade handler that bumps this (if not part of this PR, then open an issue).

Reply:
I didn't prepare the handler because the next breaking release could also be 24.3. So if we prepare some bug-fixes/optimizations, we can release them with 24.2 without voting.

Reply:
Prepared the handler in #5777, which can be renamed to 24.x latter if needed.